### PR TITLE
Upload documents from .csv, list-all-data-bundles, metadata

### PR DESCRIPTION
**Pull request by magnurud**
- Add metadata for documents
- add possibility for listing all databundles for all models
- add csv support for uploading a batch of documents
- update version and changelog

### CODE REVIEW

File: lascli/parser/models.py [hunk 1]
```diff
@@ -71,6 +74,11 @@ def list_data_bundles(las_client: Client, model_id, max_results, next_token):
     return las_client.list_data_bundles(model_id, max_results=max_results, next_token=next_token)
 
 
+def list_all_data_bundles(las_client: Client, max_results, next_token):
+    for model in list_models(las_client=las_client, max_results=max_results, next_token=next_token)['models']:
+        return las_client.list_data_bundles(model['modelId'], max_results=max_results, next_token=next_token)
```
Comment (augustak) [inline lascli/parser/models.py:79]: Is it intended that we are returning on first iteration? Maybe `yield from` was intended here?

**Review by augustak — CHANGES_REQUESTED** [review]

File: lascli/parser/models.py [hunk 2]
```diff
@@ -138,6 +147,11 @@ def create_models_parser(subparsers):
     list_data_bundles_parser.add_argument('--next-token', '-n', type=str, default=None)
     list_data_bundles_parser.set_defaults(cmd=list_data_bundles)
 
+    list_all_data_bundles_parser = subparsers.add_parser('list-all-data-bundles')
+    list_all_data_bundles_parser.add_argument('--max-results', '-m', type=int, default=None)
+    list_all_data_bundles_parser.add_argument('--next-token', '-n', type=str, default=None)
```
Comment (augustak) [inline lascli/parser/models.py:152]: Do we want to support `--next-token` and `--max-results` when listing **all** data bundles?

File: lascli/parser/models.py [hunk 3]
```diff
@@ -99,25 +72,47 @@ def create_models_parser(subparsers):
...
-    create_parser.add_argument('field_config_path', help='configuration of the fields that the model will predict')
-    create_parser.add_argument('--preprocess-config-path', '-p', help='configuration of the step before the prediction')
+    create_parser.add_argument(
+        'field_config',
+        type=path_to_json,
+        help='path to configuration of the fields that the model will predict',
+    )
+    create_parser.add_argument(
+        '--preprocess-config',
+        '-p',
```
Comment (augustak) [inline lascli/parser/models.py:82]: Should this be `type=path_to_json` as well?

**Review by augustak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 7.0.0 - 2022-02-18
+
+- Support csv files in datasets create-documents
+- Added models list-all-data-bundles
+- Support metadata for documents, datasets, models and trainings
+- Removed optional parameter --start-training from models update (Breaking change)
+
```
[the remark this answers is not in the document]
Comment (magnurud) [reply CHANGELOG.md:9]: it never existed

File: tests/util.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    max_results = [
+        ('--max-results', '5'),
+        (),
+    ]
+    next_token = [
+        ('--next-token', 'foo'),
+        (),
+    ]
+    return [ m + n for m in max_results for n in  next_token]
+
```
Comment (augustak) [inline tests/util.py:32]: Add missing new lines

**Review by augustak — CHANGES_REQUESTED** [review]

**Review by augustak — APPROVED** [review]

**Pull request merged**